fbc-integration-test/catalog-source/release/managed-workspace/release_strategy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: fbc-strategy
+  namespace: managed-release-team-tenant
+spec:
+  pipeline: fbc-release
+  bundle: quay.io/scoheb/pipeline-fbc-release:latest
+  policy: fbc-policy
+  serviceAccount: release-service-account
+  params:
```

Comment:
Take a look in https://github.com/hacbs-release/enablements/blob/main/fbc/production/release_strategy.yaml and add the missing params here.

Reply:
this all irrelevant and removed, since we are running on RHTAP stage so CRs should be deployed on Stage backend